superset-frontend/cypress-base/cypress/e2e/chart_list/filter.test.ts

```diff
@@ -41,7 +41,7 @@ describe('Charts filters', () => {
   });
 
   it('should allow filtering by "Type" correctly', () => {
-    setFilter('Type', 'Area Chart (legacy)');
```

Comment:
I'm not entirely sure what the issue was here, i.e., when I renamed this to `Time-series Area Chart (legacy)` the test failed, even though the entry exists in the dropdown. 

My hunch is, because these are listed alphabetically, that the new position was below the fold which was problematic for the Cypress test.